block_test.go

```diff
@@ -60,6 +62,73 @@ func TestSetCompactionFailed(t *testing.T) {
 	testutil.Ok(t, b.Close())
 }
 
+func TestOpenBlock_ChunkCorrupted(t *testing.T) {
+	for name, test := range map[string]struct {
+		corrFunc func(f *os.File) // Func that applies the corruption.
+		expErr   error
+	}{
+		"invalid header size": {
+			func(f *os.File) {
+				err := f.Truncate(1)
+				testutil.Ok(t, err)
+			},
+			errors.New("invalid chunk header in segment 0: invalid size"),
+		},
+		"invalid magic number": {
+			func(f *os.File) {
+				magicChunksSize := 4
```

Comment:
I don't see any problem on exposing `chunks.MagicChunksSize` so that we don't need to duplicate it for the test. The same for the other bits below.

Reply:
OK,  Thanks again.